onnxruntime/python/tools/quantization/README.md

```diff
@@ -1,5 +1,30 @@
 # Quantization tool Overview
-This tool supports quantization of an onnx model. quantize() takes a model in ModelProto format and returns the quantized model in ModelProto format.
+This tool supports 8 bit linear quantization of an onnx model. quantize() takes a model in ModelProto format and returns the quantized model in ModelProto format.
+Today ORT does not guarantee support for E2E model quantization, meaning all ONNX ops do not have support for 8 bit data types therefore only the supported ops in the model are quantized. For rest of the ops inputs are reconverted to FP32.
```

Comment:
>all ONNX ops do not have support for 8 bit data types [](start = 73, length = 53)

not all ONNX ops have support for 8 bit ...?

Reply:
That is true... We have added explicit support for ops like Convolution and Matmul and then there are other data type agnostic ops like Transpose, Reshape, Identity etc which are inherently supported. However other ops like Add, LSTM, BatchNorm etc... do not have 8 bit support yet. It was decided that we add this support based on as needed basis... .i.e when we have a model or a scenario which demands it we will go ahead and add it...

Reply:
I mean should the string "all ONNX ops do not have support for 8 bit data types" be replaced with "not all ONNX ops have support for 8 bit data types"

Reply:
I do not see a big difference between the 2 wordings... both seem to convey the same message. Let me know if you think this will really make a difference and I will update it... otherwise I prefer to check this in as is...